Tasks/AzureWebPowerShellDeployment/task.loc.json

```diff
@@ -13,7 +13,7 @@
   "version": {
     "Major": 1,
     "Minor": 0,
-    "Patch": 34
+    "Patch": 35
```

Comment:
Can you make sure you bump all of the tasks that consume the common module? I don't see AzurePowerShell. I want to make sure when 103 rolls out customers are in a predictable state wrt which version of the common module their task carries.